lib/puppet/type/cisco_bgp.rb

```diff
@@ -255,6 +256,12 @@ def dot_to_big(dot_str)
     newvalues(:true, :false, :default)
   end # property shutdown
 
+  newproperty(:enforce_first_as) do
+    desc "Enable/Disable"
```

Comment:
Please add a bit more description about this property like we have for the other global properties.